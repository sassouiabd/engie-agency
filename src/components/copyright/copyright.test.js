import React from "react";
import { render, cleanup } from "@testing-library/react";
import Copyright from "./copyright.component";

describe("Copyright component", () => {
  afterEach(cleanup);

  it("match snapshot reference", () => {
    const { asFragment } = render(<Copyright />);

    expect(asFragment(<Copyright />)).toMatchSnapshot();
  });
});
