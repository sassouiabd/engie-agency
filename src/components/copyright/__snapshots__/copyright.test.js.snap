// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright component match snapshot reference 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
  >
    Copyright Â© 
    <a
      class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
      href="https://www.engie.fr/"
    >
      engie-agency
    </a>
     2021.
  </p>
</DocumentFragment>
`;
